mobingi/sesha3/sesha3.go

```diff
@@ -0,0 +1,34 @@
+package sesha3
+
+import (
+	"github.com/moul/gotty-client"
+	"github.com/pkg/errors"
+)
+
+type SeshaClientInput struct {
+	URL string
+}
+
+type Creds struct {
+	client *gottyclient.Client
+}
+
+func New(in *SeshaClientInput) (ret *Creds, err error) {
```

Comment:
@biosugar0 Can you remove return value names? For example, `ret` and `err`. So they will be consistent with the other functions.